.travis.yml

```diff
@@ -16,6 +16,9 @@ env:
   - TEST_NO_FUSE=1 TEST_VERBOSE=1 TEST_SUITE=test_go_expensive
   - TEST_NO_FUSE=1 TEST_VERBOSE=1 TEST_SUITE=test_sharness_expensive
 
+before_install:
+  - sudo apt-get install -y librados-dev
```

Comment:
Won't work on osx - https://travis-ci.org/ipfs/go-ipfs/jobs/451847996

Reply:
fix it in today's pr